site/src/pages/RoadmapPage/Year.tsx

```diff
@@ -168,7 +168,7 @@ const Year: FC<YearProps> = ({ yearIndex, data }) => {
       {showContent && (
         <>
           <hr />
-          <div className="year-accordion-content">
+          <div className={`year-accordion-content year-accordion-content-length-${data.quarters.length}`}>
```

Comment:
class names with numbers like this feels like code smell but we'll have to discuss more in-person